tutorials/Tutorial16_Document_Classifier_at_Index_Time.ipynb

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
- Can we put in the intial description text on the top some more use case motivation? Users should understand easily in what cases that might help (tagging sentiment, tagging topics, using this for filtering ...)
- Maybe we can also show an example in the bottom for using an indexing pipeline with a docclassifier node?
- How much more effort would it be to add an example for zero-shot classification? This might make this tutorial more exciting and make others aware of the capabilities

Reply:
done: Makes much more sense with zero shot

Reply:
done: makes much more sense with zero shot classification.